### DISCUSSION TITLE
`buf beta price|stats` feedback

### DISCUSSION THREAD

**Discussion by plaflamme** [bufbuild/buf#1994] [General]
Some useful things these commands could provide:

* provide a breakdown by package / module / file / etc
* provide the full list of messages / enums / methods being considered (`beta price` sometimes spits out a large number, which we can't easily cross-check)
* provide machine-readable output like `json` so these commands can be more easily integrated in other tools (say to report into cost attribution tools)

**Comment by bufdev** [comment]
Thanks for the feedback! `buf beta price|stats` was a quick and dirty implementation, but we can definitely improve it. We'll take all of this into account and see what we can do.

Of note: both commands are extremely simple, so if you're bored, feel free to get into them and send us a PR :-)
